cypress/e2e/accessibility.cy.ts

```diff
@@ -1,7 +1,8 @@
+import { then } from "@shellygo/cypress-test-utils/assertable";
```

Comment:
I'm not sure it's a good thing to know the internal folder structure of the library...

Reply:
it's not a folder structure, it's an entry point

Reply:
Why not re export it from the index file? This way you can rename the file in the future if you want...

Reply:
1. it is separate from the CypressHelper
2. because of the issue with webpack

Reply:
This will be fixed using rollup in the lib.